code/modules/projectiles/ammo_boxes/magazine_boxes.dm

```diff
@@ -735,3 +735,297 @@
 
 /obj/item/ammo_box/magazine/deagle/super/highimpact/ap/empty
 	empty = TRUE
+
+
+//-----------------------S&W Revolver Speed Loaders Box-----------------------
+/obj/item/ammo_box/magazine/snw
+	name = "\improper speed loaders box (S&W .38 x 12)"
```

Comment:
\improper is for when the first letter is capitalized but it's not a proper noun